gpii/node_modules/packagekit/nodepackagekit/nodepackagekit_test.js

```diff
@@ -155,6 +155,17 @@ var fluid = require("universal"),
         return matchPkg;
     };
 
+    pkTests.testPerformActionFailure = function() {
```

Comment:
Just in case, here's the archive of the note:

http://lists.idrc.ocad.ca/pipermail/architecture/2016-April/004051.html

It typically prevents me from committing badly formatted code a few times a week, totally worth it.